src/test/java/com/conveyal/gtfs/storage/RecordExpectation.java

```diff
@@ -96,4 +96,12 @@ public RecordExpectation clone() {
         copy.doubleExpectation = this.doubleExpectation;
         return copy;
     }
+
+    @Override public String toString() {
```

Comment:
I prefer annotations to be on the line above. Isn't that how we write them elsewhere?